src/controllers/SlideController.ts

```diff
@@ -0,0 +1,24 @@
+import { Response } from 'express'
+import { CreatePresentationRequest } from '../interfaces/SlideInterface'
+import SlideService from '../services/SlideService'
+
+const createPresentation = async (
+  req: CreatePresentationRequest,
+  res: Response
+) => {
+  try {
+    const { link, title, isPrivate, isLive } = req.body
+    const presentation = await SlideService.createPresentation({
+      link,
+      title,
+      isPrivate,
+      isLive
+    })
+
+    res.status(200).send({ message: 'success', presentation })
+  } catch (error) {
+    res.status(400).send({ error: 'bad request' })
+  }
+}
```

Comment:
check how @Zayan-Alaraishy is handling the errors here https://github.com/CA-G12/mern-presently.io/pull/10